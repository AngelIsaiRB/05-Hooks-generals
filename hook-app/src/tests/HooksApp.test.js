import React from 'react'
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';

describe('test de el archivo hooksApp', () => {
    
    
    test('should regresar el snapshot con hola mundo', () => {
        const wrapper = shallow(<HooksApp/>);    
        expect(wrapper).toMatchSnapshot();
    })
    
})
