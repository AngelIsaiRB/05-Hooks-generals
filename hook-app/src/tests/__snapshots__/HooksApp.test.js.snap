// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test de el archivo hooksApp should regresar el snapshot con hola mundo 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
